examples/widgets/scrollbar.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:intl/intl.dart';
+import 'package:flutter/material.dart';
+
+class ScrollbarApp extends StatefulComponent {
+  ScrollbarApp({ this.navigator });
+
+  final NavigatorState navigator;
+
+  ScrollbarAppState createState() => new ScrollbarAppState();
+}
+
+class ScrollbarAppState extends State<ScrollbarApp> {
+  int _itemCount = 20;
+  double _itemExtent = 50.0;
+  ScrollbarPainter _scrollbarPainter = new ScrollbarPainter();
+
+  Widget _buildMenu(BuildContext context) {
+    var dd = new NumberFormat("00", "en_US");
+    return new ScrollableList<int>(
+      items: new List.generate(_itemCount, (i) => i),
+      itemExtent: _itemExtent,
+      itemBuilder: (_, i) => new Text('Item ${dd.format(i)}', style: Theme.of(context).text.title),
+      scrollableListPainter: _scrollbarPainter
+    );
+  }
+
+  Widget build(BuildContext context) {
+    Widget scrollable = new Container(
+      margin: new EdgeDims.symmetric(horizontal: 6.0), // TBD 6.0 should be based on _kScrollbarThumbWidth
+      child: new Center(
+        shrinkWrap: ShrinkWrap.both,
+        child: new Container(
+          width: 80.0,
+          height: _itemExtent * 5.0,
+          child: _buildMenu(context)
+        )
+      )
+    );
+
+    return new Scaffold(
+      toolBar: new ToolBar(center: new Text('Scrollbar Demo')),
+      body: new Container(
+        decoration: new BoxDecoration(backgroundColor: Theme.of(context).primarySwatch[50]),
+        padding: new EdgeDims.all(12.0),
+        child: new Center(child: new Card(child: scrollable))
```

Comment:
I was trying to keep the application boilerplate separate from the example's point, which is to demo the Material scrollbar painter.  II have to admit (also) that I find deeply nested constructions somewhat hard to read and to manage.